pyknos/mdn/mdn.py

```diff
@@ -81,7 +81,7 @@ def __init__(
             raise NotImplementedError
 
         # Constant for numerical stability.
-        self._epsilon = 1e-2
+        self._epsilon = 1e-4
```

Comment:
👍

Reply:
we should add a test in `sbi` to make sure this really fixes the issue.

Reply:
Agreed, I'll create an issue